gradle-remote

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
It would be great to add an intro to this file that explains what this file does and why. It can be inferred from the very short script, but knowing what for would be useful. (which I should have done for some of the new scripts in the scripts subfolder as well)